data/charts/chartstorage.json

```diff
@@ -3,9 +3,12 @@
         "URL": "https://tinyurl.com/y2r7tw4l"
     },
     "grades": {
-        "URL": "https://tinyurl.com/y5u9btfn"
+        "URL": "https://tinyurl.com/yxnlq8tf"
     },
     "attendance": {
         "URL": "https://tinyurl.com/yxmv9sdj"
+    },
+    "class_average_between_6_semesters": {
+        "URL": "https://tinyurl.com/ys8o9p4p"
```

Comment:
When I open this URL, I see a static image. Just out of curiosity, I want to know how this works. Can you redirect me to some part of the code or documentation where I can understand this?